Services/TermsOfService/classes/History/class.ilTermsOfServiceAcceptanceHistoryGUI.php

```diff
@@ -131,7 +132,7 @@ protected function addUserAutoComplete() : void
 
         if ($this->http->wrapper()->query()->has('term')) {
             $query = ilUtil::stripSlashes(
-                $this->http->wrapper()->query()->retrieve('term', $this->refinery->kindlyTo()->string())
+                $this->http->wrapper()->query()->retrieve('term', $this->refinery->kindlyTo()->string())  // PHP8-Review: Parameter #1 $a_str of static method ilUtil::stripSlashes() expects string, mixed given
```

Comment:
This is a `False Positive`. The `kindlyTo->string()` transformation always returns a string.